service_capacity_modeling/interface.py

```diff
@@ -13,6 +13,17 @@
 from pydantic import BaseModel
 from pydantic import Field
 
+
+class ExcludeUnsetModel(BaseModel):
+    def dict(self, *args, **kwargs):
+        kwargs["exclude_unset"] = True
```

Comment:
Can we check if the exclude_unset param was explicitly passed and is False, and if so don't exclude it?